src/main/java/at/hannibal2/skyhanni/api/CollectionApi.kt

```diff
@@ -101,7 +101,7 @@ object CollectionApi {
 
     @HandleEvent
     fun onItemAdd(event: ItemAddEvent) {
-        if (event.source == ItemAddManager.Source.COMMAND) return
+        if (event.source == ItemSources.Source.COMMAND) return
```

Comment:
no i dont like the name used here. either ItemSource.COMMAND or ItemManager.Source.COMMAND.